src/storage/txn/scheduler.rs

```diff
@@ -1333,6 +1342,17 @@ impl Scheduler {
             cids,
             cb_ctx
         );
+
+        for cid in &cids {
+            let ctx = self.cmd_ctxs.get_mut(cid).unwrap();
+            ctx.get_snapshot_timer.take();
+            ctx.processing_timer = Some(
```

Comment:
I prefer adding this to the process_read and process_write to record the real process time. Here still contains the pending time.

Btw, maybe we still need to record the pending time too.

Reply:
How to define pending here?

Reply:
see https://github.com/pingcap/tikv/blob/master/src/coprocessor/endpoint.rs#L219

Reply:
For each command in scheduler, we have `command_timer`, `latch_wait_timer`, `get_snapshot_timer`, and `processing_timer`, is the overhead too expensive?